LibraBFT/Impl/Consensus/BlockStorage/BlockTree.agda

```diff
@@ -54,8 +54,8 @@ insertBlockE block bt = do
       nothing → Left fakeErr
       (just parentBlock) → (do
         parentBlock' ← addChild parentBlock blockId
-        let bt'  = bt & btIdToBlock ∙~ Map.insert (block ^∙ ebParentId) parentBlock' (bt ^∙ btIdToBlock)
-        pure ( (bt' & btIdToBlock ∙~ Map.insert blockId (LinkableBlock∙new block) (bt' ^∙ btIdToBlock))
+        let bt' = bt & btIdToBlock ∙~ Map.insert (block ^∙ ebParentId) parentBlock' (bt ^∙ btIdToBlock)
+        pure (  (bt' & btIdToBlock ∙~ Map.insert blockId (LinkableBlock∙new block) (bt' ^∙ btIdToBlock))
```

Comment:
There are more parens than necessary (and inconsistent with the Haskell code) but otherwise looks good.

Reply:
Those parens are necessary - probably a precedence thing for our lens viz-a-viz product type.

Reply:
Ah, I see that there are more parens than SHOULD be necessary, but indeed these are necessary unless and until someone fixes the precedences, which I don't suggest doing now.